nodebuilder/pruner/flags.go

```diff
@@ -0,0 +1,36 @@
+package pruner
+
+import (
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+)
+
+var (
+	archivalModeFlag = "archival-mode"
```

Comment:
personally think explicitly calling it "pruning" or "prune-mode" is clearer, than archival. To me, archival suggests the opposite (everything is archived here, so NOT pruned and guaranteed)